src/Microsoft.ML.FastTree/BinFile/BinFinder.cs

```diff
@@ -2,12 +2,13 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System;
-using System.Threading;
+using Microsoft.ML.Runtime;
 using Microsoft.ML.Runtime.Data;
 using Microsoft.ML.Runtime.Internal.Utilities;
+using System;
+using System.Threading;
 
-namespace Microsoft.ML.Runtime.FastTree.Internal
+namespace Microsoft.ML.Trainers.FastTree.Internal
```

Comment:
>Trainers [](start = 23, length = 8)

I don't see any mentions of internal bits in #1318. It's neither estimator /trainer/estimator.
Is this renaming blessed by higher priests?
I'm not against it, I'm just curious is this right way to do or not.

Reply:
@TomFinley mentioned that we should not fragment namespaces like this one. 
I do want to make sure I got this right, hence doing just one rename on this PR. 

cc @Zruty0

Reply:
For internal code, I am not very particular as to which namespace it belongs to. So, I think I would not advocate for this change, but if it's already done, let's keep it. 

---
In reply to: [227566886](https://github.com/dotnet/machinelearning/pull/1347#discussion_r227566886) [](ancestors = 227566886)